<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>thePhoto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/thephoto.svg</normaloff>:/icons/thephoto.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>748</width>
         <height>178</height>
        </rect>
       </property>
       <widget class="QLabel" name="imageLabel">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>97</width>
          <height>18</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">ImageGoesHere</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="imageNameLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="noItemsInLibFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="criticalIcon">
             <property name="text">
              <string notr="true">icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="noImagesLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>There are no images in your library.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Manage Library</string>
           </property>
           <property name="icon">
            <iconset theme="configure">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionManage_Library"/>
    <addaction name="actionStart_Slideshow"/>
    <addaction name="separator"/>
    <addaction name="actionConnect_to_Phone"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_to_100"/>
   <addaction name="actionFit_to_Window"/>
   <addaction name="actionDelete"/>
   <addaction name="actionStart_Slideshow"/>
  </widget>
  <action name="actionConnect_to_Phone">
   <property name="icon">
    <iconset theme="smartphone" resource="icons.qrc">
     <normaloff>:/icons/smartphone.svg</normaloff>:/icons/smartphone.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Event Mode</string>
   </property>
  </action>
  <action name="actionManage_Library">
   <property name="icon">
    <iconset theme="configure" resource="icons.qrc">
     <normaloff>:/icons/configure.svg</normaloff>:/icons/configure.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage Library</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset theme="zoom-in" resource="icons.qrc">
     <normaloff>:/icons/zoom-in.svg</normaloff>:/icons/zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset theme="zoom-out" resource="icons.qrc">
     <normaloff>:/icons/zoom-out.svg</normaloff>:/icons/zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_to_100">
   <property name="icon">
    <iconset theme="zoom-100">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom to 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionFit_to_Window">
   <property name="icon">
    <iconset theme="zoom-fit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
  </action>
  <action name="actionStart_Slideshow">
   <property name="icon">
    <iconset theme="media-playback-start" resource="icons.qrc">
     <normaloff>:/icons/media-playback-start.svg</normaloff>:/icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Start Slideshow</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete" resource="icons.qrc">
     <normaloff>:/icons/edit-delete.svg</normaloff>:/icons/edit-delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about" resource="icons.qrc">
     <normaloff>:/icons/help-about.svg</normaloff>:/icons/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit" resource="icons.qrc">
     <normaloff>:/icons/application-exit.svg</normaloff>:/icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
